### ISSUE TITLE
pokesnipers close

### ISSUE THREAD

**Issue by zenis12** [5andr0/PogoLocationFeeder#261]
hmm, i press on target on pogo it opens pokesnipers, but when it says looking it close right away.. i have everything on same folder.. what is the problem? i only can snipe manually copying the coords
help <i

**Comment by zenis12** [comment]
solved,, its working on 1.4 , 1.5 im getting that problem.
another thing its says i can remove the confirmation on settings while i snipe. how can i turn that off

**Comment by thisisfizz** [comment]
what is the meaning of 1.4,1.5? do u mean the version of the sniper?

**Comment by zenis12** [comment]
the version of pogo.. 1.4 works with pokesnipers
1.5 not working, crash when it says looking for pokemon

**Comment by zenis12** [comment]
its 0.1.5 instead

**Comment by zenis12** [comment]
and 0.1.4

**Comment by RevinTrevin** [comment]
Yeah I'm having the same issue on 0.1.5 where once you click the red bulls-eye to open up Pokesniper, the Pokesniper console window opens for a split second and then closes.
